pkg/controller/model/utils.go

```diff
@@ -53,33 +44,9 @@ func getDashNamespaces(cr *v1alpha1.Grafana) string {
 
 }
 func prometheusInfo(cr *v1alpha1.Grafana) (host string, port int32) {
-	host = PrometheusServiceName
-	if cr.Spec.PrometheusServiceName != "" {
-		host = cr.Spec.PrometheusServiceName
-	}
-	if cr.Spec.DataSourceConfig != nil &&
-		DatasourceType(cr) != operator.DSTypeCommonService {
-		host = "localhost"
-	} else if cr.Spec.DataSourceConfig != nil &&
-		cr.Spec.DataSourceConfig.CommonServiceDSConfig != nil &&
-		cr.Spec.DataSourceConfig.CommonServiceDSConfig.ServiceName != "" {
-		host = cr.Spec.DataSourceConfig.CommonServiceDSConfig.ServiceName
-
-	}
-
-	port = PrometheusPort
-	if cr.Spec.PrometheusServicePort != 0 {
-		port = cr.Spec.PrometheusServicePort
-	}
-	if cr.Spec.DataSourceConfig != nil &&
-		DatasourceType(cr) != operator.DSTypeCommonService {
-		port = 9096
-	} else if cr.Spec.DataSourceConfig != nil &&
-		cr.Spec.DataSourceConfig.CommonServiceDSConfig != nil &&
-		cr.Spec.DataSourceConfig.CommonServiceDSConfig.ServicePort != 0 {
-		port = cr.Spec.DataSourceConfig.CommonServiceDSConfig.ServicePort
-
-	}
+	// return OCP prometheus host and ports
+	host = "localhost"
```

Comment:
is this OCP prometheus location fixed AND documented?

Reply:
as per ibm-grafana-ocpthanos-proxy repo(https://github.com/IBM/ibm-grafana-ocpthanos-proxy) readme file
The address ibm-grafana-ocpthanos-proxy should listen on. Default value: 127.0.0.1:9096,
so host is considered as localhost for ocp monitoring mode and port is 9096